man/tpm2_akparse.8.in

```diff
@@ -39,6 +39,11 @@ file inputFile, and output the key into file akKeyFile.
 .B tpm2_akparse
 parse the algorithm and key values in TPM2B_PUBLIC struct which input via
 file inputFile, and output the key into file akKeyFile.
+.PP
+The output file will start with a 2 byte algorithm identifier followed by one
+or more public key parts. Each public key part will start with a 2 byte
+length header followed by the actual data. Both header fields are stored in
+big endian byte order.
```

Comment:
If this is our own file format and subject to our whims, we may want to use the files.h routines for plopping a version header at the beginning. Looking at akparse, we would just want a routine for sending it to disk (versioning it along the way) and a routine for pulling it off disk)

Reply:
I don't think specifying this format is appropriate for the manpage, we want this file to be opaque in nature.

Reply:
Replaced by a note about the opaque nature of the output file.

I don't know which of the other tools use this format and take it as input any more. The command was the one that got me closest to completing the task of extracting the public key for use by external commands.

It does seem it is a custom format not explicitly mentioned in the spec though. That's why I thought documenting it would be sensible. Having a tool that "parses" one blob into another one doesn't seem so helpful. At least for users trying to get some format they can use outside of the TPM.